com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -827,15 +827,24 @@ public bool StartServer()
 
             Initialize(true);
 
-            var result = NetworkConfig.NetworkTransport.StartServer();
 
-            IsServer = true;
-            IsClient = false;
-            IsListening = true;
+            var result = NetworkConfig.NetworkTransport.StartServer();
```

Comment:
can we just kill `result` local variable entirely? (also below as well)
and turn if checks into `if (NetworkConfig.NetworkTransport.StartServer() == false)` etc.?